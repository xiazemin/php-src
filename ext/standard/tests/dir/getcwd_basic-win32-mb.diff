003+ string(116) "/Users/didi/PhpstormProjects/c/php-src/ext/standard/tests/dir/私はガラスを食べられますgetcwd_basic"
003- string(%d) "%s%e私はガラスを食べられますgetcwd_basic"