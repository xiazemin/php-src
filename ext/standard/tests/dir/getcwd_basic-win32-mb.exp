*** Testing getcwd() : basic functionality ***
string(%d) "%s"
string(%d) "%s%e私はガラスを食べられますgetcwd_basic"
===DONE===