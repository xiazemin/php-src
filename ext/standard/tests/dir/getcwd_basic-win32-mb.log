
---- EXPECTED OUTPUT
*** Testing getcwd() : basic functionality ***
string(%d) "%s"
string(%d) "%s%e私はガラスを食べられますgetcwd_basic"
===DONE===
---- ACTUAL OUTPUT
*** Testing getcwd() : basic functionality ***
string(38) "/Users/didi/PhpstormProjects/c/php-src"
string(116) "/Users/didi/PhpstormProjects/c/php-src/ext/standard/tests/dir/私はガラスを食べられますgetcwd_basic"
===DONE===
---- FAILED
