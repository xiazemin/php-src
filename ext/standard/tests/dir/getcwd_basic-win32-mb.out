*** Testing getcwd() : basic functionality ***
string(38) "/Users/didi/PhpstormProjects/c/php-src"
string(116) "/Users/didi/PhpstormProjects/c/php-src/ext/standard/tests/dir/私はガラスを食べられますgetcwd_basic"
===DONE===